person.rb

```diff
@@ -24,7 +24,7 @@ def correct_name
     @name
   end
 
-  private
+  privateb
```

Comment:
- [ ] Kindly fix this type here to be `private` instead of `privateb`.
- [ ] Please make sure the that `Person` class has `@rentals` property in its constructor and has an `add_rental` method implemented (similar to the one in the `Book` class).